### PR TITLE
ADD: Swagger dependencies and configuration

### LINKED ISSUE #31: Integrate Swagger to document API endpoints

### PR DESCRIPTION
**Pull request by TitusCln**
closes #31

### CODE REVIEW

**Comment by TitusCln** [comment]
@oscarSantoyo  please i've already implement basic swagger documentation

File: projects/src/main/java/com/sbs/projects/ProjectControllerImpl.java [hunk 1]
```diff
@@ -28,72 +31,87 @@
...
 
+    @Override
     @GetMapping(value = "/projects", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Gets all projects created and Sprints if withSprints it's set to true")
     public Iterable<ProjectDTO> getAllProjects(@RequestParam(name = "withSprints", defaultValue = "false") Boolean withSprints) {
         return projectService.getAll(withSprints);
     }
 
     @GetMapping(value = "/projects/{projectId}")
+    @ApiOperation(value = "Gets the Project with the specified projectId")
```
Comment (oscarSantoyo) [inline projects/src/main/java/com/sbs/projects/ProjectControllerImpl.java:42]: As a good practice, all methods from an interface should be marked as Override

**Review by oscarSantoyo — CHANGES_REQUESTED** [review]
Hey @TitusCln I tried out this endpoint and I'm getting XML response
http://localhost:8090/projects/1?withSprints=true

Not sure if all of them have the same issue, could you review it please.

**Comment by oscarSantoyo** [comment]
Good Job, just left a pair of comments.

**Comment by TitusCln** [comment]
@oscarSantoyo  i've already fix the comments

**Review by oscarSantoyo — APPROVED** [review]
good job!! :1st_place_medal:

**Pull request merged**